src/Tickets/Blocks/Tickets/app/editor/container/template.js

```diff
@@ -51,7 +53,18 @@ const TicketsContainer = ({
 			'event-tickets'
 		);
 	} else if (!hasCreatedTickets) {
-		messages.title = __('Add a ticket to get started.', 'event-tickets');
+		if (! hasRecurrenceRules) {
+			// translators: %s is the post type name.
+			messages.title = sprintf(
+				_x( 'Create single tickets for this %s. <a class="helper-link" href="https://evnt.is/manage-tickets" target="_blank" rel="noopener noreferrer">Learn more about ticket management</a>',
+					'event-tickets'
+				),
+				postType
```

Comment:
Besides what @lucatume mentioned, the translated string should not contain the HTML for the link itself. 
https://wordpress.stackexchange.com/questions/165357/how-to-make-a-text-with-hyperlink-translatable-in-wordpress

Take a look at the Piecemeal approach here. But to avoid doing `dangerouslySetInnerHTML` which we should. Go with two separate translations.

So Split the translation to avoid `dangerouslySetInnerHTML`.

Reply:
Thanks @lucatume. This change was applied